### PR TITLE
zygo site simplificaitons

### PR DESCRIPTION
**Pull request by guybedford**
To go with https://github.com/zygo-io/zygo-blog/pull/2.

### CODE REVIEW

File: public/routes.json [hunk 1]
```diff
@@ -1,3 +1,9 @@
 {
-  "/": "zygo-blog/routes.json"
+  "component": "app/page.jsx!",
+
+  "/:pid": "zygo-blog/views/article.jsx!",
+  "/:cid": {
+    "/": "zygo-blog/views/article-list.jsx!",
+    "/:pid": "zygo-blog/views/article.jsx!"
+  }
```
Comment (guybedford) [inline public/routes.json:8]: I like this because I can immediately tell all my routes just by looking at the router json.

File: zygo.json [hunk 2]
```diff
@@ -6,11 +6,14 @@
...
-      "overview": "/post/overview",
-      "faq": "/post/faq"
+      "getting started": "/zygo",
+      "tutorials": "/tutorials/",
+      "docs": "/docs/",
+      "overview": "/overview",
+      "faq": "/faq",
+      "github": "https://github.com/zygo-io"
     }
   },
```
Comment (guybedford) [inline zygo.json:18]: The `db_server` still needs to be added here.

**Comment by guybedford** [comment]
Thanks, I'll pair with you today on working this through.

File: zygo.json (hunk 2 above)
Comment (timlind) [inline zygo.json:18]: Not sure if this sort of data should really be in the zygo.json? It should be wrappable with the installable routes.

**Pull request merged**